src/cbc_sdk/platform/alerts.py

```diff
@@ -1794,6 +1796,51 @@ def set_group_by(self, field):
 
         return grouped_alert_search_query
 
+    def export(self):
+        """
+        Starts the process of exporting Alerts from the organization in CSV format.
+
+        This starts an asynchronous job.  Wait for completion of the job and
+        then use one of the get results methods on the Job.
+
+
+        Example:
+            >>> job = cb.select(Alert).add_criteria("type", ["CB_ANALYTIC", "WATCHLIST"]).export()
+            >>> job.await_completion().result()
+            >>> csv_report = job.get_output_as_string()
+
+
+        Required Permissions:
+            org.alerts (READ)
+            jobs.status (READ) to get the results
+
+        Returns:
+            Job: The asynchronous job that will provide the export output when the server has prepared it.
+        """
+        request = self._build_request(0, -1, add_rows=False)
+        request["format"] = "CSV"
```

Comment:
Does this have an option for JSON output? Or is it just hard-wired to always expect CSV for now?

Reply:
CSV only at this stage.  JSON is on the roadmap.
Have modified the method to accept export_format as an optional parameter, defaults to CSV.

Reply:
Cool.